docker/wordpress/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM wordpress:5.5
+
+ARG WPGRAPHQL_VERSION=v0.13.3
+ARG WPGATSBY_VERSION=v0.4.17
+ARG YOAST_GRAPHQL_VERSION=v4.8.0
+ARG WPGRAPHQL_GUTENBERG_VERSION=v0.3.4
+ARG WPGRAPHQL_ACF_VERSION=v0.3.5
+
+LABEL maintainer="duffn@outlook.com" 
+
+# Set timezone, install XDebug, PHP Composer, WP-CLI
+RUN echo 'date.timezone = "UTC"' > /usr/local/etc/php/conf.d/timezone.ini \
+  && rm -rf /var/lib/apt/lists/* \
+  && pecl install xdebug \
+  && docker-php-ext-enable xdebug \
+  && docker-php-ext-install pdo_mysql \
+  && curl -Ls 'https://raw.githubusercontent.com/composer/getcomposer.org/fc4099e0ac116a1c8f61fffaf6693594dda79d16/web/installer' | php -- --quiet \
+  && chmod +x composer.phar \
+  && mv composer.phar /usr/local/bin/composer \
+  && curl -O 'https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar' \
+  && chmod +x wp-cli.phar \
+  && mv wp-cli.phar /usr/local/bin/wp
+
+
+COPY install-plugin.sh /usr/local/bin/install-plugin
+
+# WP GraphQL
+RUN install-plugin wp-graphql wp-graphql/wp-graphql ${WPGRAPHQL_VERSION}
```

Comment:
@jasonbahl and then we install the plugins here, using just the release tarball for now. I tried getting composer to work here in the image build, but it didn't pan out because of user permissions issues I couldn't reconcile. thinking maybe we can run composer in the wp-cli process, where the user permissions are set up better for installing plugins?

Reply:
@acao you might be able to install a plugin outside of wp-cli and then activate it with wp-cli